include/droptables.h

```diff
@@ -24,11 +24,14 @@ typedef union {
 } Droptable;
 static_assert(sizeof(Droptable) == 0x20);
 
-extern const Droptable gUnk_0800137C[];
-extern const Droptable gUnk_0800143C[];
-extern const Droptable gUnk_080015BC[];
-extern const Droptable gUnk_0800161C[];
+extern const Droptable gDroptablesEnemies[];
+extern const Droptable gDroptablesAreas[];
+extern const Droptable gDroptablesObjects[];
 extern const Droptable gUnk_0800191C[];
-extern const Droptable gUnk_08001A1C[];
+extern const Droptable gDroptablesModifiers[];
+
+#define gDroptableModifierNone gDroptablesModifiers[0]
```

Comment:
Could this be in caps and without the g to avoid confusion? Also I feel like using an enum and referring to them like gDroptablesModifiers[NO_SHELLS] might be better.